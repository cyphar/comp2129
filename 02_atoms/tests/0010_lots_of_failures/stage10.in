LOAD a.a
PLAYFROM ENDa
PLAYFROM -2
PLAYFROM 1000lol
PLAYFROM 0test
PLAYFROM
PLAYFROM END testing
PLAYFROM 10000000000000
STAT
