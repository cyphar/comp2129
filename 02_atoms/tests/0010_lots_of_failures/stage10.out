Game Loaded

Invalid Turn Number

Invalid Turn Number

Invalid Turn Number

Invalid Turn Number

Missing Argument

Too Many Arguments

Game Ready
Red's Turn

Player Red:
Grid Count: 0

Player Green:
Grid Count: 0

